ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/ExecutionStep.java

```diff
@@ -68,6 +70,10 @@ default S build(PlanBuilder planBuilder) {
     return build(planBuilder, extractPlanInfo(new PlanInfoExtractor()));
   }
 
+  default S build(PlanBuilder planBuilder, PlanInfo planInfo, boolean addToTopology) {
```

Comment:
Added this flag to control when an execution step is added to the Streams topology